### DISCUSSION TITLE
GRID tool

### DISCUSSION THREAD

**Discussion by wmdhosting** [tinymce/tinymce#8325] [Ideas]
Can you replicate
https://imperavi.com/article/

this GRID workflow.. 
![Snag_22abb98](https://user-images.githubusercontent.com/76176291/205677602-b4de9ec7-8061-4531-94a7-3810ab416338.png)

**Comment by lostkeys** [comment]
Thanks for the feedback! To help us better understand and prioritize your request, can you fill us in on the following details. If you don't want to do it publicly, feel free to submit your feedback over at our [roadmap website](https://www.tiny.cloud/roadmap/)

1. Why is the requested feature important for your app/service?
2. What is the consequence of not having the requested feature?
3. How important would you rate the requested feature choosing from nice-to-have, important or critical?

Thanks,
Fredrik – Product Manager
